### ISSUE TITLE
zod-openapi access to registry

### ISSUE THREAD

**Issue by WildEgo** [honojs/middleware#146]
Hi, I was wondering if it would be possible to export the `registry` on the `zod-openapi` middleware, I'd like to create a component like the one below so I can represent that parts of my API require a Bearer token in order to be accessed.
```ts
    const bearerAuth = registry.registerComponent(
      'securitySchemes',
      'bearerAuth',
      {
        type: 'http',
        scheme: 'bearer',
        bearerFormat: 'JWT',
      }
    );
```

Thanks in advance and I'd like to praise this amazing package, never found such a nice "framework" to work with!

**Comment by yusukebe** [comment]
Hi @WildEgo, thank you for your kind words!

> I was wondering if it would be possible to export the `registry` from the `zod-openapi` middleware,

I originally made the `registry` private because I wanted to ensure users added routes via `app.openapi()` and not directly. I believed there was no need for direct access to the `registry`. However, in light of your situation, it seems appropriate to make it public.

I believe a simple solution would be to make the `registry` property public though there is possibility to be renamed such as `openAPIRegistry`. What are your thoughts on this?

**Comment by WildEgo** [comment]
Hi @yusukebe , for some reason I really can't quote your comment, that sounds fair enough yeah, if that was possible I'd be glad to use it, adding custom "components" by themselves would really help and I think this would be a more then good enough solution.

**Comment by yusukebe** [comment]
Hi @WildEgo !

I've released `hono/zod-openapi@0.3.0` includes the feature to make the register a public property. You can try it.
